src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+@import '../blocks/stars';
+@import '../utils/variables';
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: block;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  background-color: $white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px;
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    display: block;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 40px;
```

Comment:
[Code Style Rules:](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#css) Be consistent with your margins (Add only top or bottom, but not both) To avoid potential margin collapse
![image](https://github.com/user-attachments/assets/9297583b-bf2b-457e-9bee-cff593a57d24)